cmake/configs/default.cmake

```diff
@@ -175,7 +175,7 @@ hunter_config(damageproto VERSION 1.2.1)
 hunter_config(dbus VERSION 1.10.0-hunter-4)
 hunter_config(debug_assert VERSION 1.3)
 hunter_config(dest VERSION 0.8.0-p4)
-hunter_config(dlib VERSION 19.6-p2)
+hunter_config(dlib VERSION 19.8-p0)
```

Comment:
Doesn't belong here, can you please rebase?

Reply:
I'll take a look in > 1 hour.